scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/utils/DataFrameUtils.scala

```diff
@@ -64,23 +64,29 @@ object DataFrameUtils {
       var labelMum: Int = 0
       val inputList = new java.util.ArrayList[Float]()
       val arr = if (featureColumn != null || labelColumn != null) {
+        featureColumn.foreach(f => inputList.add(r.getAs[Float](f)))
```

Comment:
why don't you set featureColumn to a required and none-empty array？

Reply:
Could support file without column names in this case, e.g. [diabetes-csv-origin](https://github.com/intel-analytics/BigDL/pull/3714/files?authenticity_token=8hwWLoxoodBuVEmmOywMdkJX2DoVW%2ByignPa%2BvIwMUfsQxer5HScpXtPclxaMKgBQHc69NGgDUS%2FiLNfyV2U5A%3D%3D&file-filters%5B%5D=.scala&hide-deleted-files=true#diff-d029c0ace7eacbc2457a9a9256f2164e5850f4507831a3bb306abb6c18f7a278)

Reply:
What will happen if the featureColumn is an empty array or the featureColumn is null

Reply:
the last column would be label and others would be features